upgrade_test.py

```diff
@@ -813,10 +813,14 @@ def _add_sla_credentials_to_stress_commands(self, workloads_with_sla: list):
 
         roles = []
         service_level_shares = self.params.get("service_level_shares")
+        # OSS Scylla version does not support "shares" option for Service Level
+        if service_level_shares and not self.db_cluster.nodes[0].is_enterprise:
```

Comment:
wasn't it better to just `return` in this command `if not self.db_cluster.nodes[0].is_enterprise` ?